src/type.cpp

```diff
@@ -1503,6 +1511,8 @@ bool ArrayType::IsBoolType() const { return false; }
 
 bool ArrayType::IsConstType() const { return child ? child->IsConstType() : false; }
 
+bool ArrayType::IsCompleteType() const { return GetBaseType()->IsCompleteType(); }
```

Comment:
We have an unsized arrays which I think are also incomplete types.

Reply:
I think that, generally, unsized arrays should not be considered incomplete types unless they are the only members of a struct.